docker-compose.yml

```diff
@@ -998,6 +1008,7 @@ services:
       ConnectionStrings__DatabaseConnection: Server=tcp:mssql,1433;Initial Catalog=link-tenant;Persist Security Info=False;User ID=sa;Password=${LINK_DB_PASS};MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=True;Connection Timeout=30;
       Serilog__WriteTo__0__Args__uri: http://loki:3100
       Telemetry__EnableTelemetry: true
+      Telemetry__EnableTracing: true
```

Comment:
Should check deployed App Configs in DEV and TEST to make sure this is enabled. Also, should inform DevOps that this is necessary in all environments.